src/main/java/io/openbpm/control/view/processdefinition/DeleteProcessDefinitionView.java

```diff
@@ -56,6 +59,9 @@ public void onInit(final InitEvent event) {
     @Subscribe
     protected void onBeforeShow(BeforeShowEvent event) {
         deleteProcessInstancesCheckBox.setValue(true);
+
+        long countByProcessDefinitionId = processInstanceService.getCountByProcessDefinitionId(processDefinitionId);
```

Comment:
We should also add an explanation why the checkbox cannot be enabled. Camunda displays the following message in the Cockpit:
```
There are 1 running process instance which belong to this deployment. In order to delete this deployment it is necessary to enable the option Cascade flag.
```